components/ui/Tag.jsx

```diff
@@ -0,0 +1,9 @@
+function Tag(props) {
+  return (
+    <button className="m-auto h-fit rounded-4.75 bg-light-grey px-2 text-grey">
```

Comment:
- [ ] All tags have same height, it could be better to define it on the component
- [ ] There is a specified width of a tag though it differs respective to the tag content
- [ ]  The padding on x is not the correct value defined on figma